packages/itwin/ec3-widget/src/components/CreateAssemblyComponent.scss

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+.ec3w-create-assembly-step {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  .ec3w-assembly-list {
+    height: calc(100vh - 248px);
+    overflow: auto;
+    .ec3w-assembly-expandable-block {
+      margin-top: 8px;
+      .iui-svg-icon {
+        display: flex;
+        flex-direction: row-reverse;
```

Comment:
Try not to override or target iTwin UI styles. It becomes very difficult to maintain and can break between versions.

Reply:
https://github.com/iTwin/iTwinUI/wiki/Version-conflicts
```Avoid using internal APIs and CSS class names, as these can often break between versions. iui- and _iui prefixed classes are considered an implementation detail and you should not target them; instead, use your own custom class names or data attributes.```

Reply:
keeping this as it is, as discussed, added comment.